src/demux/tsdemuxer.ts

```diff
@@ -182,6 +183,10 @@ class TSDemuxer implements Demuxer {
 
     this._videoTrack = TSDemuxer.createTrack('video') as DemuxedVideoTrack;
     this._videoTrack.duration = trackDuration;
+    this.videoIntegrityChecker =
+      this.config.handleMpegTsVideoIntegrityErrors === 'skip'
```

Comment:
@mstyura Why only check the integrity of video PES packets? 

Would it be better or even simpler to check the integrity of all packets?

```suggestion
    this.integrityChecker =
      this.config.handleMpegTsIntegrityErrors === 'skip'
```